scripts/migration/migrate_date_last_request_for_current_users.py

```diff
@@ -0,0 +1,34 @@
+"""
+Used to add users date_last_request, value set equal to date_last_login.
+"""
+
+import sys
+import logging
+from modularodm import Q
+from website.app import init_app
+from website import models
+from scripts import utils as script_utils
+from framework.transactions.context import TokuTransaction
+
+logger = logging.getLogger(__name__)
+
+
+def main(dry=True):
+    init_app(routes=False)
+    users = models.User.find(Q('date_last_request', 'eq', None))
+    logger.warn('All active users will have a date_last_request field added and set equal to date_last_login.')
```

Comment:
Should this be a `.info`? `.warn` and `.error` are usually reserved for when something is going wrong.